dataframely/columns/datetime.py

```diff
@@ -284,6 +293,7 @@ def __init__(
                 the formatting language used by :mod:`polars` datetime ``round`` method.
                 For example, a value ``1h`` expects all datetimes to be full hours. Note
                 that this setting does *not* affect the storage resolution.
+            time_zone: The timezone that datetimes in the column must have.
```

Comment:
Would it make sense to add a sentence here to say what string formats we accept for time zones?

Reply:
Updated to give a better explanation and gave examples too